.github/workflows/c-cpp.yml

```diff
@@ -12,15 +12,15 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-18.04
+          - ubuntu-22.04
     steps:
       - name: Install dependencies (Ubuntu)
         if: runner.os == 'Linux'
-        run: >-
+        run: |
           sudo apt-get update
-
-          sudo apt-get install -y curl libcurl4-openssl-dev cmake gstreamer1.0-plugins-base
-          libgstreamer1.0-dev python3-pip python3-setuptools
+          # remove libunwind-*-dev from the runner to avoid conflicts with a dependency of libgstreamer1.0-dev
```

Comment:
How does this affect a user that's trying to build from source? Will they need to also follow these steps to uninstall `libunwind-*-dev`?